package.json

```diff
@@ -8,10 +8,14 @@
   },
   "devDependencies": {
     "@types/node": "^12.12.21",
+    "@yarnpkg/sdks": "^3.0.0-rc.43",
     "github-label-slack": "workspace:*",
     "lerna": "^6.5.1",
     "prettier": "^1.19.1",
-    "typescript": "^4.9.5"
+    "typescript": "^5.0.4"
+  },
+  "resolutions": {
+    "typescript": "^5.0.4"
```

Comment:
why is this resolution necessary?

Reply:
It's not, but it ensures that nothing uses the wrong version.